pkg/arvo/sur/graph-store.hoon

```diff
@@ -0,0 +1,69 @@
+/-  *post
+|%
+++  mop
+  |*  [key=mold value=mold]
+  |=  ord=$-([key key] ?)
+  |=  a=*
+  =/  b  ;;((tree [key=key val=value]) a)
+  ?>  (check-balance:((ordered-map key value) ord) b)
+  b
+::
++$  mark          term
```

Comment:
mark is already term; this is superfluous

Reply:
Good point.